jdl/exporters/jdl-exporter.spec.ts

```diff
@@ -103,7 +104,10 @@ describe('jdl - JDLExporter', () => {
             let jdlObject;
             beforeEach(() => {
               jdlObject = new JDLObject();
-              jdlObject.addApplication(new JDLApplication({ config: { [CLIENT_FRAMEWORK]: NO_CLIENT_FRAMEWORK } }));
+              const jdlApplication: JSONApplication = {
```

Comment:
Same confusion about the relevance: shouldn't it be `jdl/models/jdl-application.ts`